modules/tile-converter/src/i3s-server/controllers/index-controller.ts

```diff
@@ -12,9 +12,16 @@ const I3S_LAYER_PATH = process.env.I3sLayerPath || '';
  */
 export async function getFileNameByUrl(url: string): Promise<string | null> {
   const extensions = ['json', 'bin', 'jpg', 'jpeg', 'png', 'bin.dds', 'ktx2'];
-  const FULL_LAYER_PATH = path.join(process.cwd(), I3S_LAYER_PATH);
+  const absolutePath = /^[^.]+/.exec(I3S_LAYER_PATH);
+  let filePath = process.cwd();
+  let fileURLToPath = url;
+  if (absolutePath) {
+    filePath = '';
+    fileURLToPath = '';
```

Comment:
We discard input `url` here. It seems for me that this will work only for the root index json.

Reply:
`I3S_LAYER_PATH` is the path to the http service root. What about path to the dataset? It might not be in the root. The might be multiple datasets in the root folder. 
Also there are resources - node pages, nodes. geometry, textures. All these resources should be available through http requests.

Reply:
Corrected algorithm to have access to different resources